src/modules/create-market/components/create-market-form-review/create-market-form-review.jsx

```diff
@@ -5,16 +5,24 @@ import { augur } from 'services/augurjs'
 // import BigNumber from 'bignumber.js'
 
 import { formatEtherEstimate, formatEtherTokensEstimate } from 'utils/format-number'
+import { EXPIRY_SOURCE_GENERIC } from 'modules/create-market/constants/new-market-constraints'
 
 import Styles from 'modules/create-market/components/create-market-form-review/create-market-form-review.styles'
 import StylesForm from 'modules/create-market/components/create-market-form/create-market-form.styles'
 
 export default class CreateMarketReview extends Component {
+  static formatResolutionSource(expirySourceType, expirySource) {
+    if (expirySourceType === EXPIRY_SOURCE_GENERIC) {
+      return 'Outcome will be determined by news media'
+    }
+
+    return `Outcome will be detailed on public website: ${expirySource}`
+  }
 
   static propTypes = {
     newMarket: PropTypes.object.isRequired,
     universe: PropTypes.object.isRequired,
-    meta: PropTypes.object
+    meta: PropTypes.object,
```

Comment:
Do you currently have the real-time style + es linting setup in your editor?

That last comma isn’t explicitly necessary.